lighthouse-core/test/lib/stack-packs-test.js

```diff
@@ -0,0 +1,169 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* eslint-env jest */
+
+const lighthouseStackPacksDep = require('lighthouse-stack-packs');
+const stackPacksLib = require('../../lib/stack-packs.js');
+const defaultConfig = require('../../config/default-config.js');
+const Config = require('../../config/config.js');
+
+function getAuditIds() {
+  const config = new Config(defaultConfig);
+  return config.audits.map(a => a.implementation.meta.id);
+}
+
+describe('stack-packs lib', () => {
+  it('packs without detectors', () => {
+    const result = lighthouseStackPacksDep
+      .filter(p => !stackPacksLib.stackPacksToInclude.find(p2 => p2.packId === p.id))
+      .map(p => p.id);
+    expect(result).toMatchInlineSnapshot(`
+      Array [
+        "drupal",
+      ]
+    `);
+  });
+});
+
+// These tests summarize the contents of the lighthouse-stack-packs package.
+describe('lighthouse-stack-packs dep', () => {
+  it('snapshot packs', () => {
+    expect(lighthouseStackPacksDep.map(p => p.id)).toMatchInlineSnapshot(`
+      Array [
+        "wordpress",
+        "drupal",
+        "react",
+        "angular",
+        "amp",
+        "magento",
+      ]
+    `);
+  });
+
+  // Keys should only be added, not removed.
```

Comment:
Is this instructions for someone updating the snapshot or a description of the test?

Reply:
for  updating the snapshot. moved it down a bit.